pages/404.tsx

```diff
@@ -0,0 +1,28 @@
+import type { NextPage } from 'next'
+import Image from 'next/image'
+import Link from 'next/link'
+import error404 from '../public/404/error-404.svg'
```

Comment:
Replace this import to use recommended NextJS image method.

Reply:
@RenatoSiqueira I thought this was the recommended NextJS method: https://nextjs.org/docs/basic-features/image-optimization#local-images

Can you please explain in more datils what would be done to solve this issue?